source/trackers.json

```diff
@@ -1072,7 +1078,7 @@
         "adjust.net.in": "adjust",
         "adjust.world": "adjust",
         "apptrace.com": "adjust",
-        "adobe.net": "adobe_experience_cloud",
+        "adobe.net": "adobe_audience_manager",
```

Comment:
Is not it used by many their services?

Reply:
`adobe.com` in whotracksme base refers to `adobe_audience_manager`